AutomaticInterface/AutomaticInterface/InterfaceBuilder.cs

```diff
@@ -103,21 +94,16 @@ public string Build()
                 cb.AppendLine("#nullable enable");
             }
 
-            foreach (var usg in usings)
-            {
-                cb.AppendLine(usg);
-            }
-
-            cb.AppendLine("");
-
             cb.AppendLine($"namespace {nameSpaceName}");
             cb.AppendLine("{");
 
             cb.Indent();
 
             cb.AppendAndNormalizeMultipleLines(classDocumentation);
 
-            cb.AppendLine($"[GeneratedCode(\"AutomaticInterface\", \"\")]");
+            cb.AppendLine(
+                "[global::System.CodeDom.Compiler.GeneratedCode(\"AutomaticInterface\", \"\")]"
```

Comment:
This was a choice - it seemed simpler and more concise to fully qualify this too, since otherwise there's only be one using in the interface file, `using System.CodeDom.Compiler;`